scripts/test-renode.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+
+SCRIPT_SRC=$(realpath ${BASH_SOURCE[0]})
+SCRIPT_DIR=$(dirname $SCRIPT_SRC)
+TOP_DIR=$(realpath $SCRIPT_DIR/..)
+if [[ "${BASH_SOURCE[0]}" != "${0}" ]]; then
+	echo "You must run this script, rather then try to source it."
+	echo "$SCRIPT_SRC"
+	exit 1
+fi
+
+source $SCRIPT_DIR/build-common.sh
+
+init
+
+RENODE_BIN=${RENODE_BIN:-renode}
+RENODE_FOUND=false
+
+if [ -x "$RENODE_BIN" ]; then
+	RENODE_FOUND=true
+fi
+
+if command -v "$RENODE_BIN" 2>&1 1>/dev/null; then
+	RENODE_FOUND=true
+fi
+
+if ! $RENODE_FOUND; then
+	# Download prebuilt renode Release if none is currently installed
+	conda install -c antmicro -c conda-forge renode
+	RENODE_BIN=$CONDA_PREFIX/bin/renode
+fi
+
+case $CPU in
+	vexriscv | picorv32)
+		;;
+	*)
+        echo "Simulating the current configuration is not supported in Renode yet - skipping the test"
```

Comment:
Indenting here looks a bit weird.

Reply:
It was caused by mixed tabs & spaces. Fixed now.